### ISSUE TITLE
Need quotations in upload script

### ISSUE THREAD

**Issue by dreese** [Instabug/Instabug-iOS#17]
https://github.com/Instabug/Instabug-iOS/blob/master/Instabug.bundle/Instabug_dsym_upload.sh#L63

Need to change this line:
```
DSYM_UUIDs=$(dwarfdump --uuid ${DSYM_PATH} | cut -d' ' -f2)
```
to
```
DSYM_UUIDs=$(dwarfdump --uuid "${DSYM_PATH}" | cut -d' ' -f2)
```

When the scheme build type has a space in it, such as "Ad Hoc", this line will cause an error.

**Comment by msoliman** [comment]
Thanks for pointing this out. I'll make sure that we fix it in our next release.

**Comment by dreese** [comment]
Thanks!

**Comment by msoliman** [comment]
Fixed in 2e7a6e13e3adc7427b944f194e6ed41ff8901f57

**Comment by dreese** [comment]
Awesome. Thanks!
